packages/sdk/governance/src/proposals.ts

```diff
@@ -287,37 +289,44 @@ export class ProposalBuilder {
   getRegistryAddition = (contract: CeloContract): string | undefined =>
     this.registryAdditions[stripProxy(contract)]
 
-  isRegistered = (contract: CeloContract) =>
+  isRegistryContract = (contract: CeloContract) =>
     RegisteredContracts.includes(stripProxy(contract)) ||
     this.getRegistryAddition(contract) !== undefined
 
-  buildFunctionCallToExternalContract = async (
+  buildCallToExternalContract = async (
```

Comment:
Does this need to be renamed? It would avoid breaking changes if we kept it the same.